SDWebImageAVIFCoder/Classes/SDImageAVIFCoder.m

```diff
@@ -5,6 +5,7 @@
 //  Created by lizhuoli on 2018/5/8.
 //
 
+#include <alloca.h>
```

Comment:
One small question ? `alloca` seems allocate the memory buffer on stack. Does AVIF raw pixel buffer smaller enough to safelly allocate on stack but not heap ?

I'm not so professional on image processing, just a little curious about this. SDWebImage have many codec integration, previous every of them using the heap allocated buffer not on stack.

Reply:
I found that [non-main thread just have 512KB stack](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Multithreading/CreatingThreads/CreatingThreads.html) in iOS, so it is dangerous to use `alloca` to allocate, even for dummy data.

I pushed a commit that replaces `alloca` to `calloc`.